src/kiwixapp.cpp

```diff
@@ -41,28 +41,20 @@ KiwixApp::KiwixApp(int& argc, char *argv[])
         return;
     }
 
-    bool loaded = false;
-    bool installed = false;
 #if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
-    loaded = m_qtTranslator.load(QLocale(), "qt", "_",
-                        QLibraryInfo::location(QLibraryInfo::TranslationsPath));
+    QString path = QLibraryInfo::location(QLibraryInfo::TranslationsPath);
 #else
-    loaded = m_qtTranslator.load(QLocale(), "qt", "_",
-                        QLibraryInfo::path(QLibraryInfo::TranslationsPath));
+    QString path = QLibraryInfo::path(QLibraryInfo::TranslationsPath);
 #endif
-    if (loaded) {
-        installed = installTranslator(&m_qtTranslator);
-        if (!installed) {
-            // ignore for now
-        }
+    bool success = false;
+    success = loadAndInstallTranslations(m_qtTranslator, "qt", path);
+    if (!success) {
+        // ignore for now
     }
```

Comment:
Why don't you move this into `loadAndInstallTranslations()` and change the latter's return type to `void`?

Reply:
My particular style (very common in Go) is that the calling function checks the return value and then decides how to handle the error (e.g. log something). Moving the success check into the called function would mean the called function has to decide what to do on failure, which I see as a bit of a general anti-pattern, especially if the goal is to maintain consistent handling during failures.

I solved this by simply ignoring the return value from `loadAndInstallTranslations()`, let me know if that works.